software/controller/lib/hal/hal_stm32.cpp

```diff
@@ -121,9 +121,19 @@ void HalApi::Init() {
   init_PCB_ID_pins();
   LEDs.initialize();
   SystemTimer::singleton().initialize(CPUFrequency);
-  /// \TODO: fault somehow if this returns false
-  [[maybe_unused]] bool buffer_size_sufficient = adc.initialize(CPUFrequency);
-  buzzer.initialize(CPUFrequency);
+
+  // \TODO: previous implementation of buzzer used a 0.8 scaling factor to produce max volume
```

Comment:
I don't know if calling it an actuator is necessarily **wrong**. Actually, some googling has brought me to pages that claim that loudspeakers are actually classified as "sound actuators" :)

Reply:
https://www.electronics-tutorials.ws/io/io_8.html#:~:text=Loudspeakers%20are%20audio%20sound%20transducers,original%20input%20signal%20as%20possible.